lib/sufia/import/generic_file_translator.rb

```diff
@@ -0,0 +1,42 @@
+module Sufia::Import
+  # Imports a Sufia 6.0-exported GenericFile into a Sufia PCDM GernericWork and FileSet
+  class GenericFileTranslator
+    # @param Hash settings needed to pass through to builder
+    #        @attr sufia6_user      - User name for accessing the sufia 6 fedora
+    #        @attr sufia6_password  - Password for accessing the sufia 6 fedora
+    #        @attr import_binary    - (default true) Import the binary content from fedora
+    def initialize(settings)
+      # TODO: work builder doesn't need settings; just file builder
+      @work_builder = WorkBuilder.new(settings)
+      @file_set_builder = FileSetBuilder.new(settings)
+    end
+
+    # loop through the files found in import_dir and map each GenericFile to a Sufia.primary_work and FileSet
+    # @param import_dir a directory containing export files, e.g.: generic_file_[id].json
+    # @return list of imported files
+    def import(import_dir)
+      # TODO: error instead of return
+      return unless Dir.exist?(import_dir)
+      # turn import_dir into a list of actual filenames
+      files = Dir.glob(File.join(import_dir, '*')).select { |f| exported_gf?(f) }
+      files.each { |f| import_file(f) }
```

Comment:
Okay I think I see where we're not seeing eye-to-eye; see my comment from a couple days ago: https://github.com/projecthydra/sufia/issues/2141#issuecomment-253327681

I am now thinking it sounds like some of the work I've done here should move to the exe, which will pull the file list and pass just that to the translator. will rearrange.